### PR TITLE
Add event logging for database

### PR DESCRIPTION
**Pull request by ting-c**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -2,12 +2,9 @@
 // main.js is basicly what makes the bot run. There are more comments throughout that help descripe some elements.
 //
 const Discord = require('discord.js');
-
-const mongoose = require('mongoose'); 
-const fs = require('fs');
-
 const config = require('dotenv').config();
-const { prefix, token, roles, MongoDB } = require('./config.json');
+const { token } = require('./config.json');
```
Comment (j-dogcoder) [inline main.js:6]: Shouldn't this still be `const { prefix, token, roles, MongoDB } = require('./config.json');`?

File: main.js [hunk 2]
```diff
@@ -2,12 +2,9 @@
 // main.js is basicly what makes the bot run. There are more comments throughout that help descripe some elements.
 //
 const Discord = require('discord.js');
-
-const mongoose = require('mongoose'); 
-const fs = require('fs');
```
Comment (j-dogcoder) [inline main.js:7]: We still need fs

**Review by j-dogcoder — CHANGES_REQUESTED** [review]

File: main.js (hunk 2 above)
Comment (ting-c) [reply main.js:7]: Made the two changes in main.js

File: main.js (hunk 1 above)
Comment (ting-c) [reply main.js]: Made the two changes in main.js

**Pull request merged**